features/support/check/checkCookieExists.js

```diff
@@ -0,0 +1,25 @@
+const assert = require('assert').strict;
+
+/**
+ * Checks if an cookie does, or does not, exist on the page. 
+ * Throws an error if the cookie name is not defined in the document.
+ * @param {String} cname Cookie name to check for.
+ * @param {String} not The string "not" to negate the check (cookie value should not match).
+ */
+module.exports = async function(cname, not) {
+ 
+  var cookieNameFound;
```

Comment:
Change this up to `let` to make it block scoped.  If you're curious, you can read more about the difference between [var and let here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let)

```suggestion
  let cookieNameFound;
```